modules/update-service-graph-data.adoc

```diff
@@ -0,0 +1,87 @@
+[id="update-service-graph-data_{context}"]
+= Mirroring the {product-title} Update Service graph data
+
+The {product-title} Update Service requires a graph-data container image, from which the Update Service retrieves information about channel membership and blocked update edges.
+
+.Prerequisites
+
+* An link:https://cloud.redhat.com/openshift/install/pull-secret[{product-title} pull secret].
+* A pull secret for the local registry.
+
+.Procedure
+
```

Comment:
AFAIK, the graph data has not been released as an image yet, so we have to build the image manually. I think the procedure for building the image is required.